modules/client/test/integration/addresslistVoting-client/methods.test.ts

```diff
@@ -281,6 +299,70 @@ describe("Client Address List", () => {
     });
   });
 
+  describe("Plugin installation", () => {
+    it("Should prepare the installation of a token voting plugin", async () => {
+      const networkSpy = jest.spyOn(JsonRpcProvider, "getNetwork");
+      networkSpy.mockReturnValueOnce({
+        name: "goerli",
+        chainId: 31337,
+      });
+      const ctx = new Context(contextParamsLocalChain);
+      const ctxPlugin = ContextPlugin.fromContext(ctx);
+      const client = new AddresslistVotingClient(ctxPlugin);
+      const { dao } = await buildAddressListVotingDAO(
+        repoAddr,
+        VotingMode.VOTE_REPLACEMENT,
+      );
+      const installationParams:
+        AddresslistVotingPluginPrepareInstallationParams = {
+          settings: {
+            votingSettings: {
+              supportThreshold: 0.5,
+              minParticipation: 0.5,
+              minDuration: 7200,
+              minProposerVotingPower: BigInt(1),
+              votingMode: VotingMode.STANDARD,
+            },
+            addresses: [TEST_WALLET_ADDRESS],
+          },
+          daoAddressOrEns: dao,
+        };
+      const steps = client.methods.prepareInstallation(installationParams);
+      for await (const step of steps) {
+        switch (step.key) {
+          case PrepareInstallationStep.PREPARING:
+            expect(typeof step.txHash).toBe("string");
+            expect(step.txHash).toMatch(/^0x[A-Fa-f0-9]{64}$/i);
```

Comment:
you need to verify it but if I am not mistaken the values of step are always the same, because the test wallet is the same and after each test the nonce gets reset. thus doing things result always in the same hash. But I am no 100% sure

Reply:
Mmmm, not sure the nonce gets reseted, because the chain only gets deployed one time and for ach test we aree creating a dao, proposal or whatevceer is needed